projects/sbb-esta/angular-public/src/lib/dropdown/dropdown-trigger.directive.ts

```diff
@@ -92,6 +93,33 @@ export const DROPDOWN_SCROLL_STRATEGY_FACTORY_PROVIDER = {
   useFactory: SBB_DROPDOWN_SCROLL_STRATEGY_FACTORY
 };
 
+const panelPositionNames = {
```

Comment:
As an alternative to the `as ConnectedPosition`, you can define the type of panelPositionNames:
```
const panelPositionNames: { [key: string]: ConnectedPosition } = {
...
```